xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -105,6 +102,8 @@
   private static final String EDS_RESOURCE = "cluster-load-assignment.googleapis.com";
   private static final String VERSION_1 = "42";
   private static final String VERSION_2 = "43";
+  private static final String LISTENER_RESOURCE =
```

Comment:
Ah, why don't you just put it together with other xDS resource constant?    That's how code gets messy......